internal/api/mfa_test.go

```diff
@@ -124,7 +124,7 @@ func (ts *MFATestSuite) TestEnrollFactor() {
 			user, err := models.FindUserByEmailAndAudience(ts.API.db, "test@example.com", ts.Config.JWT.Aud)
 			ts.Require().NoError(err)
 
-			token, err := generateAccessToken(ts.API.db, user, nil, &ts.Config.JWT)
+			token, _, err := generateAccessToken(ts.API.db, user, nil, &ts.Config.JWT)
```

Comment:
at some point in the future perhaps we can consider merging token, expiresAt and other relevant fields returned into a struct of sorts